### PR TITLE
Add a test to show slow resolution (recursion problem?)

### PR DESCRIPTION
**Pull request by kpanic**

### CODE REVIEW

**Comment by lorenzosinisi** [comment]
Very good catch! The PR to optimize this is now live https://github.com/lorenzosinisi/retex/pull/20

**Comment by lorenzosinisi** [comment]
Fixed in https://github.com/lorenzosinisi/retex/pull/20, will publish a new version of Retex and update all deps tomorrow 👍 Thanksss

File: test/integration_test.exs [hunk 1]
```diff
@@ -173,4 +173,38 @@ defmodule NeuralBridge.IntegrationTest do
...
              |> Map.fetch!(:inferred_facts)
   end
+
+  test "recursion bug?" do
+    rules = [
+      NeuralBridge.Rule.new(
+        id: 1,
+        given: """
+        Birth's place is equal "Italy"
+        Birth's place is equal "Albania"
```
Comment (lorenzosinisi) [inline test/integration_test.exs:183]: This could also become: `Birth's place is in ["Albania", "Italy"]` 👍

Comment (kpanic) [reply test/integration_test.exs:183]: Sanskrit already supports that. Nice 💯

**Pull request closed without merging**